lib/kiev/shoryuken/store_request_details.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Kiev
+  module Shoryuken
+    class StoreRequestDetails
+      WRAPPED = "wrapped"
```

Comment:
Didn't find any usage of this constant. Probably redundant.

Reply:
Yes. Leftovers from Sidekiq's feature that doesn't appear to exist in Shoryuken.